### DISCUSSION TITLE
Add more milk recipes

### DISCUSSION THREAD

**Discussion by Unrepentant-Atheist** [cataclysmbn/Cataclysm-BN#2804] [General]
### Is your feature request related to a problem? Please describe.

Game has raw milk -> milk recipe. That's it. But there's pasteurized and UHT milk too. Not rocket science how to do them.

### Describe the solution you'd like

Use hotplate or something to make milk -> pasteurized milk -> UHT milk and add sinking calcium content and enjoyability. Should take less time to do than powdered milk.

**Comment by chaosvolt** [comment]
The regular milk item would be the regular pasteurized version, since making it entails processing raw milk with heat (though as the temperatures involved don't involve bringing it to a boil, it probably should either take longer than boiling water, and/or require less uses of `water_boiling_heat`).

Adding a way to make your own sealed jars or other containers of UHT would definitely be a neat alternative to powdering it though.
